CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to Suspensive libraries
+
+We welcome contribution from everyone in the community. <br/>
+All communications in this repo will be by English.
+
+> Every contributor to Suspensive libraries should adhere to our Code of Conduct.
+> <br/>Please read the [full text](./CODE_OF_CONDUCT.md) to understand what actions will and will not be tolerated.
+
+# Instructions for Logging Issues
+
+## 1. Search for Duplicates
+
+[Search the existing issues](https://github.com/suspensive/react/issues) before logging a new one.
+
+Some search tips:
+
+- _Don't_ restrict your search to only open issues. An issue with a title similar to yours may have been closed as a duplicate of one with a less-findable title.
+- Check for synonyms. For example, if your bug involves an interface, it likely also occurs with type aliases or classes.
+- Search for the title of the issue you're about to log. This sounds obvious but 80% of the time this is sufficient to find a duplicate when one exists.
+- Read more than the first page of results. Many bugs here use the same words so relevancy sorting is not particularly strong.
+- If you have a crash, search for the first few topmost function names shown in the call stack.
+
+## 2. Do you have a question?
+
+The issue tracker is for **issues**, in other words, bugs and suggestions.
+If you have a _question_, please use [Stack Overflow](https://stackoverflow.com/questions/tagged/suspensive), your favorite search engine, or other resources.
```

Comment:
I have a question here.
What's better  getting questions about libraries betweenstack overflow and GitHub issues?

Reply:
I'd like to give up stack overflow just now to maintain easily by unifying the place to ask questions

Reply:
I opened Discussion tab. Let's traffic to our GitHub Discussion.

Reply:
After remove content about stackoverflow, Let's merge this